test/unit/sorted-views.spec.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'underscore';
 
-describe('collection/composite view sorting', function() {
+describe.skip('collection/composite view sorting', function() {
```

Comment:
For this POC just commenting out this entire spec file as it is a mix between old collectionview tests and compositeview ones that no longer apply.